src/base/orgs/List.svelte

```diff
@@ -13,14 +16,26 @@
     margin: 3rem;
     display: inline-block;
   }
+  .list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
+  }
 </style>
 
+<div class="list">
 {#each orgs as org}
-  <div class="org">
-    <Link to={`/orgs/${org.address}`}>
-      <Blockies glowOnHover address={org.address} />
-    </Link>
-  </div>
+  {#await Profile.get(org.address, config)}
+    <!-- Avoid using <Loading /> here since it fills the page with loading elements -->
```

Comment:
I agree, but at the same time, now we have an intermediate state where there's no loading indicator at all, and just an empty area. Perhaps the ideal would be to have a `Profile.getMulti([string], config)` that sends the requests in parallel and only returns when all have resolved. This way we can show a single loader and also load faster.

Reply:
![image](https://user-images.githubusercontent.com/40774/124942488-f6836380-e00b-11eb-903d-ed73322ca774.png)